ui/src/services/alertHooks.js

```diff
@@ -0,0 +1,69 @@
+//@flow
+import { useAlerts } from '../containers/AlertProvider';
+import { getNodes } from './k8s/core.js';
+import { getMetalk8sV1alpha1VolumeList } from './k8s/Metalk8sVolumeClient.generated';
+import type { FilterLabels, Alert } from './alertUtils';
+import { getHealthStatus } from './alertUtils';
+
+type Query = { cacheKey: string, queryFn: Function, options?: Object };
+
+export const getNodesAlertName = (): string[] => {
+  return ['NodeAtRisk', 'NodeDegraded'];
+};
+
+export const getVolumesAlertName = (): string[] => {
+  return ['VolumeAtRisk', 'VolumeDegraded'];
+};
+
+export const getNetworksAlertName = (): string[] => {
+  return ['ControlPlaneNetworkDegraded', 'WorkloadPlaneNetworkDegraded'];
+};
+
+export const getServicesAlertName = (): string[] => {
+  return ['PlatformServicesAtRisk', 'PlatformServicesDegraded'];
+};
```

Comment:
I think a better abstraction would be to return a set of selectors instead of the name, in this way if in the futur we need to narrow down the selection of alerts we won't need to change the definition of those functions.

 ```suggestion
export const getNodesAlertSelectors = (): FilterLabels => {
  return {name: ['NodeAtRisk', 'NodeDegraded']};
};
// ...
```